templates/plugin/abrt.yml.erb

```diff
@@ -0,0 +1,28 @@
+---
+:enabled: <%= scope.lookupvar("foreman_proxy::plugin::abrt::enabled") %>
```

Comment:
We mostly use scope because in most templates the variables are not in scope. Here we do have them so IMHO it makes it clear to use them. @domcleal what do you think?